source/stylesheets/base/_typography.scss

```
@@ -2,9 +2,12 @@ html {
     font-family: $font-family-base;
 }
 
+h1, h2, h3 {
+    @include margin(0rem);
```

No need to use Bourbon’s margin mixin here. In this case, the output of the mixin is the same as just using vanilla CSS:

```suggestion
margin: 0;
```